java/vector/src/main/java/org/apache/arrow/vector/schema/TypeLayout.java

```diff
@@ -156,6 +156,11 @@ public TypeLayout visit(Binary type) {
       }
 
       @Override
+      public TypeLayout visit(ArrowType.FixedSizeBinary type) {
+        return newFixedWidthTypeLayout(dataVector(type.getByteWidth() * 8));
```

Comment:
This should just be `newVariableWidthTypeLayout()` to be consistent with Binary type?

Reply:
I think it should still be FixedWidthTypeLayout because each element is fixed-size and no offset vector is necessary

Reply:
Sorry, I meant the data type width should be 8-bit instead of type.getByteWidth() * 8.

VarBinary has 8-bit data type width and FixedBinary should be the same.

Reply:
Actually, now I read more of the doc, I am not too sure.

https://github.com/apache/arrow/blob/master/format/Schema.fbs#L225

@wesm, can you clarify the what the correct bit_width in the vector layout should be?